Update legends for panels 3B & 3C (#228)
AlexsLemonade/ScPCA-manuscript: content/100.figure-table-legends.md

A cell type summary report with information about reference sources, comparisons among cell type annotation methods, and diagnostic plots is created.
Although not shown in this panel, cell type annotations are also included in the `Processed AnnData Object` created from the `Processed SCE Object` (Figure {@fig:fig2}A).

B. Diagram of ontology-aware consensus cell type annotation performed in `scpca-nf`.
We perform pairwise comparisons among cell type annotations made by `SingleR`, `CellAssign`, and `SCimilarity`.
For each comparison, we identify the cell types' latest common ancestor based on the Cell Ontology. 

> We then identify the consensus cell type as the latest common ancestor with the fewest descendents. 

Reviewer: This is clear and concise, I like it!

> C. Example heatmap as shown in the cell type summary report comparing the consensus cell type annotations to automated annotations made with `SingleR`, `CellAssign`, and `SCimilarity`. 

Reviewer: ```suggestion
C. Example heatmap as shown in the cell type summary report comparing the consensus cell type annotations to automated annotations assigned by `SingleR`, `CellAssign`, and `SCimilarity`. 
```
I generally use the word "assign" when referring to automated tools and cell types. I feel like it's slightly more accurate?
